src/_P097_Esp32Touch.ino

```diff
@@ -318,14 +318,12 @@ void P097_setEventParams(int pin, uint16_t threshold) {
       case 8: touchAttachInterrupt(T8, P097_got_T8, threshold); break;
       case 9: touchAttachInterrupt(T9, P097_got_T9, threshold); break;
 #  if HAS_T10_TO_T14
+      case 10: touchAttachInterrupt(T10, P097_got_T10, threshold); break;
+      case 11: touchAttachInterrupt(T11, P097_got_T11, threshold); break;
+      case 12: touchAttachInterrupt(T12, P097_got_T12, threshold); break;
+      case 13: touchAttachInterrupt(T13, P097_got_T13, threshold); break;
+      case 14: touchAttachInterrupt(T14, P097_got_T14, threshold); break;
```

Comment:
Please do test on other nodes as well. (e.g. ESP32-S3)
There was a reason I did disable these when implementing ESP-IDF5.x support.
At that time it caused crashes. Could be those are now fixed, but better test before merging.